src/action_types/dashboard.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2018-present Riff Learning, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/* eslint-disable header/header */
```

Comment:
Looking at other mm action_types, they all have the `// @flow` annotation at the top, why doesn't this new file do that?

Reply:
Filed removed, N/A.